src/keynames.ts

```diff
@@ -154,29 +166,23 @@ function keyCodeToString(keycode, index) {
     piece = pieces[i];
     strrep = keyNames[piece] ?? piece;
     const className = getKeyClassName(strrep);
-
-    if (i.length === 1) {
```

Comment:
This line is clearly mad. In practice is has always been undefined so the else branch was taken. The title case version of a single character is the same as uppercasing it anyway.

Reply:
I don't think there's any need `toTitleCase("a")` will result in `"A"` via `'a'.toUpperCase() + ''.toLowerCase()`.

Reply:
Ah, fair enough. I'd missed that substring returns an empty string in that case.